src/core/server/http/cookie_session_storage.ts

```diff
@@ -139,9 +154,26 @@ export async function createCookieSessionStorageFactory<T>(
     path: basePath,
     clearInvalid: false,
     isHttpOnly: true,
-    isSameSite: false,
+    isSameSite: cookieOptions.sameSite === 'None' ? false : cookieOptions.sameSite ?? false,
   });
 
+  // A hack to support SameSite: 'None'.
+  // Remove it after update Hapi to v19 that supports SameSite: 'None' out of the box.
+  if (cookieOptions.sameSite === 'None') {
```

Comment:
Note: it should work `None ` for all evergreen browsers according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite 
We are dropping IE11 support in v7.9. It requires additional checks when backporting to the older branches!